salt/docker/init.sls

```diff
@@ -16,6 +16,7 @@ include:
         Environment="NO_PROXY={{ salt['pillar.get']('proxy:no_proxy', '') }}"
   cmd.run:
     - name: systemctl daemon-reload
+    - stateful: True
```

Comment:
The thing is nobody will be notified if, for example,  `salt['proxy:no_proxy']` changes (even when a different file has been written to disk) because of the empty output of `systemctl daemon-reload`.